store/sqlstore/upgrade.go

```diff
@@ -180,6 +190,60 @@ func upgradeDatabase(sqlStore SqlStore, currentModelVersionString string) error
 	return nil
 }
 
+func decideMaxExecutionTime(sqlStore SqlStore) int64 {
+	result := MAX_TIMEOUT_EXECUTION_TIME_MYSQL
+	if sqlStore.DriverName() == model.DATABASE_DRIVER_POSTGRES {
+		result = MAX_TIMEOUT_EXECUTION_TIME_POSTGRES
+	}
+	return result
```

Comment:
maybe a clearer and simplest option would be
```suggestion
	if sqlStore.DriverName() == model.DATABASE_DRIVER_POSTGRES {
		return MAX_TIMEOUT_EXECUTION_TIME_POSTGRES
	}
	return MAX_TIMEOUT_EXECUTION_TIME_MYSQL
```

Reply:
Yeah, I like it